template/node12/index.js

```diff
@@ -8,10 +8,15 @@ const app = express()
 const handler = require('./function/handler');
 const bodyParser = require('body-parser')
 
-// app.use(bodyParser.urlencoded({ extended: false }));
-app.use(bodyParser.json());
-app.use(bodyParser.raw());
-app.use(bodyParser.text({ type : "text/*" }));
+if (process.env.RAW_BODY === 'true') {
```

Comment:
Hey @burtonr 

I'd suggest to keep support of json and text parsers even when RAW_BODY is true.

To keep completely old behavior I suggest inverting the variable name something like NORAW_BODY and enable raw parser by default. 

jsonLimit can be also set only if MAX_JSON_SIZE is specified

Reply:
> I'd suggest to keep support of json and text parsers even when RAW_BODY is true.

I don't see the point in that change, that's the behaviour we have today?

Reply:
If I understand this correctly RAW_BODY should  just enable/disable raw parser, is not it?

If it is set to true the app will have no JSON and TEXT parsers

Reply:
The use-case was that users need to disable JSON parsing and only access a RAW body. The docs PR I hope helps with this, but I'd like @burtonr to explain the use-case better there. https://docs.openfaas.com/cli/templates/#nodejs-12-node12-of-watchdog-template

Reply:
RAW_BODY confuses me since it was TLDR. maybe ONLY_RAW name is better?

Reply:
That's right, but it's important to understand that the `bodyParser.raw()` _only_ enables the raw body for the content type that you pass in. So, this change makes it so that when a user sets the `RAW_BODY` variable to true, the raw body will be available for _all_ content types.
The idea is that a user would define a very specific function that requires the raw body, where other functions will not have this enabled and have the JSON body available as normal.